examples/mediatek/models/llm_models/modeling_common.py

```diff
@@ -757,9 +757,9 @@ def get_example_inputs(
             nt = Dim("num_token", max=num_token)
             cache_dims = tuple(({} for _ in range(2 * self.num_blocks)))
             dynamic_shapes = (
```

Comment:
I think I may ask in the previous PR - what does `dynamic_shape` do? Probably good to add a comment to explain

Reply:
Hi @cccclai, dynamic_shape is passed in to torch.export.export_for_training to indicate that the input shapes during calibration for some of the inputs may be different. After calibration, the dynamic_shape argument is not needed